packages/components/table/stories/table.stories.mdx

```diff
@@ -75,6 +74,95 @@ export const Template = ({ cellProps = () => ({}), ...args }) => {
   </Story>
 </Canvas>
 
+### CellConfig
+
+Tables are built by passing in the cellConfigs prop to the table. They represent
+the configuration for each cell within a row. In the story above, all rows are
+rendered by reading from the records directly.
+
+#### Example
+
+```jsx
+const CELL_CONFIGS = [
+  { dataKey: "name" },
+  { dataKey: "id" },
+  { dataKey: "myFacility.name" },
+  { dataKey: "level" },
+]
+
+const records = [
+  { name: "Jerry Seinfeld", id: 1, myFacility: { name: "Paris " }, level: 1 },
+  { name: "Cosmo Kramer", id: 2, myFacility: { name: "Paris " }, level: 7 },
+  { name: "Elaine Benes", id: 3, myFacility: { name: "Paris " }, level: 4 },
+  { name: "George Costanza", id: 4, myFacility: { name: "Paris " }, level: 0 },
+]
+```
+
+<table className="doc-table">
+  <thead>
+    <tr>
+      <th>Option</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>dataKey</td>
+      <td>
+        A unique identifier for the cell in a row. This can be propert of the
+        record being rendered or a nested property within the record being rendered.
+      </td>
+    </tr>
+    <tr>
+      <td>cellProps (optional)</td>
+      <td>
+        Props to be passed to the table cell when rendered. They can be any props
+        that can be passed to `Box`.
+      </td>
+    </tr>
+    <tr>
+      <td>render</td>
+      <td>
+        A function used in order to render the row. If provided, the record will
+        be passed as the only argument to the render function. If no render function
+        is provided then td will attempt to read the dataKey from the object.
```

Comment:
This got me thinking... we got backed into a corner of positional arguments in the past when we wanted to support more features in the render function. Should we change the render functions argument to be an object? Updating its interface **from:**
```
render: (rowData) => rowData.foo
```
**To**
```
render: ({ rowData }) => rowData.foo
```

Reply:
I kinda prefer the first argument being the record and if we ever take additional args it being an options object that can then be destructed.

Reply:
Oh I like that, cool!